deploymentExperiences.md

```diff
@@ -0,0 +1,6 @@
+Trying to deoploy my python django weba app on Heroku was trick and frustrating , a lot of step in the chain and you are not sure if you have done something wrong on each step of the chain to break something . 
```

Comment:
I agree that deployment can be quite frustrating. There's alot of steps and a lot to get used to. Sounds like you got quite far in the process, good job!